core/modules/user/user.module

```diff
@@ -3272,6 +3371,25 @@ function user_register_form($form, &$form_state) {
   $options = backdrop_parse_url($path);
   $options['attributes']['class'][] = 'form-cancel';
 
+  $user_login_appearance_setting = user_login_appearance_setting();
+  if ($user_login_appearance_setting != 'tabs') {
```

Comment:
Here, again, to avoid the problem in this [comment](https://github.com/backdrop/backdrop-issues/issues/6570#issuecomment-2561923371), it's best to use `user_page_is_simplified_page()` in the if statement.

```php
if (user_page_is_simplified_page()) {
 // Rest of the code
}
```

Reply:
Hmm... my suggestion does not work for the "Use links" settings, as the "Log in" link is supposed to be shown there, but it's not, since the Use links settings is different from the simplified one.